test/test_tree_2d_mhd.jl

```diff
@@ -69,7 +69,8 @@ EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "tree_2
     @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_mhd_blast_wave.jl"),
       l2   = [0.17646728395490927, 3.866230215339417, 2.4867304651291255, 0.0, 355.4562971958441, 2.359493623565687, 1.4030741420730297, 0.0, 0.029613599942667133],
       linf = [1.581630420824181, 44.15725488910748, 13.056964982196554, 0.0, 2244.875490238186, 13.07679044647926, 9.14612176426092, 0.0, 0.5154756722488522],
-      tspan = (0.0, 0.003))
+      tspan = (0.0, 0.003),
+      maxiters_coverage = 9) # For some reason, this doesn't run with maxiters < 9
```

Comment:
Why does this elixir crash with `maxiters < 9`?

Reply:
What kind of error do you get?

Reply:
`DomainError`, I think in a square root.

Reply:
All variants of `maxiters < 9` crash? Weird...

Reply:
Is it an adaptive time stepping scheme that discards the first 9 steps and if you stop after <9 iterations, you end up with an invalid solution? Would such an error even be possible?

Reply:
It's just Carpenter-Kennedy

Reply:
Then my assumption would be that it is still a "hard" IC that causes negative density/pressure values during the first few iterations. By some miracle, it will recover later though and not cause errors anymore.

Have you tried running the test manually and setting `analysis_interval=1` and `maxiters=1e5`? If I am correct, it should crash in the first few time steps - we just didn't notice it, because we didn't compute error norms during the first iterations...

Reply:
Yeah, you're right. It's the analysis callback that causes this. That makes at least a bit more sense.

Reply:
Could you explain this in the comment, please? Alternatively, you could also reduce the the analysis polydeg to avoid interpolations

Reply:
And maybe create an issue for @andrewwinters5000 - as our go-to guy for all things MHD - that this test case seems to be broken? 😬

Reply:
Well, it's not really broken - if you interpolate some values near zero, you should expect to get something negative...

Reply:
Yeah, at the beginning the initial conditions are quite harsh with a strong discontinuity in the pressure. So trying to re-interpolate to a higher `polydeg` for analysis is not a good idea.